app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -1,18 +1,21 @@
 package com.ikarabulut;
 
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
 import java.util.ArrayList;
 import java.util.List;
 
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.DisplayName;
 import static org.junit.jupiter.api.Assertions.*;
 
 
 class BoardTest {
 
     @Test
     @DisplayName("Given no arguments, a 10x10 board is created by default")
-    void setGridDefault() {
+    void setGrid_NoArgumentsPassed_10x10BoardIsGenerated() {
```

Comment:
Thanks for improving these test names. Also these test DispayNames are cool, and help me understand what condition is being tested for.

Is `@DisplayName` new in JUnit 5?

Reply:
Yes! Junit 5 added this feature. I still think my test method names are a bit redundant now but they are still following the naming conventions I am seeing out in the wild